src/pages/community.js

```diff
@@ -0,0 +1,683 @@
+  padding-left: 2rem;
+  height: 720px;
+  width: 100%;
+  margin-top: -1px;
+  margin-bottom: 0rem;
+  border-top: 1px solid ${(props) => props.theme.colors.text};
+  border-bottom: 1px solid ${(props) => props.theme.colors.text};
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column-reverse;
+    height: 100%;
+    padding-top: 2rem;
+    padding-left: 0rem;
+    padding-bottom: 2rem;
+  }
+`
+
+const SupportContainer = styled.div`
+  background: ${(props) => props.theme.colors.homeBoxPurple};
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  height: 720px;
+  width: 100%;
+  margin-top: -1px;
+  border-top: 1px solid ${(props) => props.theme.colors.text};
+  border-bottom: 1px solid ${(props) => props.theme.colors.text};
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column-reverse;
+    height: 100%;
+  }
+`
+
+const LeftColumn = styled.div`
+  margin-right: 2rem;
+  width: 100%;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    margin: auto 0rem;
+  }
+`
+
+const FeatureContent = styled(LeftColumn)`
+  padding: 6rem;
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    padding: 2rem;
+  }
+`
+
+const LeftColumnContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const IntroLeftColumn = styled(LeftColumn)`
+  padding: 6rem;
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    padding: 2rem;
+  }
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    padding: 0rem;
+  }
+`
+
+const CommunityPage = ({ data }) => {
+  const intl = useIntl()
+
+  // const heroContent = {
+  //   title: translateMessageId("page-community-title, intl),
+  //   header: translateMessageId("page-community-hero-header", intl),
+  //   subtitle: translateMessageId("page-community-hero-subtitle", intl),
+  //   image: data.stablecoins.childImageSharp.fluid,
+  //   alt: translateMessageId("page-community-hero-alt", intl),
+  //   buttons: [
+  //     {
+  //       content: translateMessageId("page-community-hero-button", intl),
+  //       path: "#",
+  //     },
+  //     {
+  //       content: translateMessageId(
+  //         "page-community-developer-support-button",
+  //         intl
+  //       ),
+  //       path: "#",
+  //       isSecondary: "isSecondary",
+  //     },
+  //   ],
+  // }
+
+  const heroContent = {
+    title: "Join the community",
+    header: "Welcome to the Ethereum community hub",
+    subtitle:
+      "The Ethereum community is home to hundreds of thousands of developers, technologists, designers, users, HODLers and enthusiasts.",
+    image: data.enterprise.childImageSharp.fluid,
+    alt: translateMessageId("page-community-hero-alt", intl),
+    buttons: [
```

Comment:
What should our primary CTAs be here? 🤔 

Should we have any? Or let users scroll down to see options?